content/faculty/honours_board.md

```diff
@@ -5,14 +5,22 @@ tags: [_nav]
 submenu_index: 4
 ---
 
+
 <ul class="nav nav-tabs" id="honours-board-tab">
-  <li class="active"><a href="#staff" data-toggle="tab">Сотрудники</a></li>
-  <li><a href="#students" data-toggle="tab">Студенты</a></li>
-  <li><a href="#sport-heroes" data-toggle="tab">«Герои спорта»</a></li>
+    <li class="active default-tab"><a href="#staff2015" data-toggle="tab">Сотрудники</a></li>
+    <li><a href="#students2015" data-toggle="tab">Студенты</a></li>
+    <li><a href="#sport-heroes2015" data-toggle="tab">«Герои спорта»</a></li>
+    <li style="float:right">
+        <select class="change-year-select">
+            <option data-display="Select" disabled>Год</option>
```

Comment:
It seems like the `option` tag isn't being used in a semantically correct way here. Use `optgroup` instead like this:

```html
<optgroup label="Год">
  <option value="2016">2016</option>
  ...
</optgroup>
```